saga-web/src/main/resources/saga-frontend/src/app/@theme/layouts/sample/sample.layout.scss

```diff
@@ -0,0 +1,179 @@
+@import '../../styles/themes';
```

Comment:
are we using sample.layout.scss anywhere? please double check

Reply:
The entire site is currently using sample.layout